### PR TITLE
HTML for Adoptadog website

### PR DESCRIPTION
**Pull request by LucasKnezevich**
I have created the html for the homepage (index), dogs, blog, and checkout pages.

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    <nav class="main-nav">
+      <a href="index.html" target="_blank">Home  /</a>
+      <a href="dogs.html" target="_blank">Dogs  /</a>
+      <a href="blog.html" target="_blank">Blog</a>
+    </nav>
+
+    <section>
+        <h2>Adoptadog Blog</h2>
+        <article class="traveling">
```
Comment (jimmynono) [inline blog.html]: You'll probably want to use the same class for each of these articles for ease in styling.

File: dogs.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        <img src="images/murphy-card.jpg" alt="Picture of Murphy, reddish-brown fur with white on snout">
+        <h3>Murphy</h3>
+        <p><b>Cost to Adopt:</b> $123.45</p>
+        <p>Doggo ipsum pupperino extremely cuuuuuute heckin clouds big ol shoober smol</p>
+        <button type="button" name="adoptButton">Adopt</button>
+      </article>
+      <article class="dog-card">
+        <img src="images/poppy-card.jpg" alt="Picture of Poppy trotting down street, white and curly fur">
+        <h3>Poppy</h3>
+        <p><b>Cost to Adopt:</b> $123.45</p>
```
Comment (jimmynono) [inline dogs.html:45]: Maybe consider using strong instead of the b tag, you can set the font weight in css

**Comment by LucasKnezevich** [comment]
Added more styling/CSS, main edits were on; positioning/format for checkout section of header, centering and sizing button for dog cards, and small adjustments to checkout form.

File: styles/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,254 @@
+/* Styling for ALL pages */
```
Comment (jimmynono) [inline styles/stylesheet.css:1]: Your CSS looks good. Just a suggestion to clean up your code a bit. Make sure you're using consistent layout for your CSS.

```
h1 {
  color: red;
}

h2 {
  color: white;
}
```

The spaces and formatting won't affect the site, but it does look a lot better, think of it as penmanship.

**Comment by LucasKnezevich** [comment]
JS folder and file created, linked to each html page.  Alerts name, breed, and cost when dog card image is clicked.  Also alerts the running cart total when the adopt button is clicked for a dog.

File: js/adoptMain.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+let cartTotal = 0;
+let dogCost = 0;
+
+function dogDescription(name, breed, cost) {
+  alert(`This is ${name}, a ${breed} breed.  $${cost} to adopt!`)
```
Comment (ClovisI) [inline js/adoptMain.js]: To be more technically precise, you could add a ; to the end of your alert(); if you want.

File: js/adoptMain.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
+//  DOG CARD IMAGES  //
+$('.dog-card').hover(function(e) {
+  $('img', this).toggleClass('.image-hover');
```
Comment (jimmynono) [inline js/adoptMain.js:3]: for the class you don't need the period, it's actually adding the class `.image-hover` instead of `image-hover`

**Pull request closed without merging**